samples/peripheral/radio_test/sample.yaml

```diff
@@ -34,3 +34,12 @@ tests:
       - nrf5340dk_nrf5340_cpunet
     platform_allow: nrf5340dk_nrf5340_cpunet
     tags: ci_build
+  sample.peripheral.radio_test.nrf5340_nrf21540.no_automatic_power:
+    build_only: true
+    extra_args: SHIELD=nrf21540_ek
+    extra_configs:
+      - CONFIG_RADIO_TEST_POWER_CONTROL_AUTOMATIC=n
```

Comment:
Here also `CONFIG_MPSL_FEM_POWER_MODEL` can be set to `n`

Reply:
This is build only test to check if sample compile well in both mode since we have there conditional compilation. I do not really see the needs to put here also `CONFIG_MPSL_FEM_POWER_MODEL`